build/policy/files.rego

```diff
@@ -58,6 +58,17 @@ deny[sprintf("Integration '%v' missing required attribute '%v'", [name, attr])]
 	some attr in (required - {key | some key, _ in item})
 }
 
+deny[sprintf("Integration '%v' references unknown software '%v' (i.e. not in 'software' object)", [name, software])] {
+	"docs/website/data/integrations.yaml" in filenames
+
+	file := yaml.unmarshal(integrations_file)
+	software_list := {software | file.software[software]}
+
+	some name, item in file.integrations
+	some software in object.get(item, ["software"], [])
```

Comment:
why not
```suggestion
	some software in item.software
```
? if item.software is undefined, `object.get(item, ["software"], [])` gives you `[]`, so `some software in []` fails, too. (I might be missing something, in a hurry)

Reply:
That's fair. Updated.